CMakeLists.txt

```diff
@@ -21,11 +21,26 @@ if(IN_OPENWRT)
     INCLUDE_DIRECTORIES("$ENV{TOOLCHAIN_DIR}/usr/include" "$ENV{TARGET_LDFLAGS}" "$ENV{TARGET_CPPFLAGS}" "$ENV{TARGET_CFLAGS}")
 endif()
 
+#do not add or build any tests of the dependencies
+set(original_run_e2e_tests ${run_e2e_tests})
+set(original_run_int_tests ${run_int_tests})
+set(original_run_unittests ${run_unittests})
+
+set(run_e2e_tests OFF)
+set(run_int_tests OFF)
+set(run_unittests OFF)
+
 add_subdirectory(azure-c-shared-utility)
```

Comment:
Since you're not including UT_tests and you don't have umock repo included, shared-util will not include the test repos and it won't find the umock.lib.

Reply:
Thanks, this was an awesome comment. I included c shared before turning off the tests for now, until I convert the tests to umock_c

---
In reply to: [145236036](https://github.com/Azure/azure-event-hubs-c/pull/21#discussion_r145236036) [](ancestors = 145236036)